lib/burndown_chart.rb

```diff
@@ -114,9 +114,10 @@ def setup(burndown_dir, board_id)
     write_data File.join(burndown_dir, burndown_data_filename)
   end
 
-  def self.plot(sprint_number, working_dir=".")
+  def self.plot(sprint_number, options)
     plot_helper = File.expand_path("../../scripts/create_burndown.py", __FILE__ )
-    system "python #{plot_helper} #{sprint_number} #{working_dir}"
+    cli_switches = options.keys.map{|k| k.dup.prepend('--')}
```

Comment:
This also breaks, when called as `trollolo burndown --plot` as the update method still expects the old `plot` signature.